app/controllers/admin/accounting/transactions_controller.rb

```diff
@@ -169,7 +169,8 @@ def settings_link
     view_context.link_to(t('menu.accounting_settings'), admin_accounting_settings_path)
   end
 
-  def sample_transaction
-    ::Accounting::Transaction.new(project: @loan, txn_date: Time.zone.today)
+  def sample_transaction(attrs = {})
+    most_likely_editable_attrs = {project: @loan, txn_date: Time.zone.today, managed: true}
```

Comment:
Above we are passing in `project_id` and supposedly expecting it to override `project`, but it's not clear to me if that is reliable. Can we make both use `project`?

Reply:
Separately, I'm wondering what the purpose of using the sample method here is, as opposed to just adding `managed: true` to the `merge` in `create`. Thoughts?

Reply:
it looks like sample_transaction was there to DRY things up between new and sync. I'm updating it to not use sample_transaction, and have each endpoint method have their own 'sample transaction' so they are each responsible for knowing what their sample txn should be. I agree the merge overwrite is confusing (I looked up the docs to make sure the overwrite is in the right order, but putting that into comments seems clunky).